x-pack/plugins/alerting/server/routes/rule/apis/mute_all/mute_all_rule.ts

```diff
@@ -36,7 +31,7 @@ export const muteAllRuleRoute = (
       },
       validate: {
         request: {
-          params: paramSchema,
+          params: muteAllRuleRequestParamsSchemaV1,
         },
         response: {
```

Comment:
Previously, when we did this versioning we ended up also updating the documentation. I don't know if we will do another passing but I noticed 403 is missing from the `response`. Maybe you could update that here? 🙏

Reply:
Maybe `400` and `404` also. [I'm thinking of this PR.](https://github.com/elastic/kibana/pull/192616)

Reply:
If I'm not mistaken, there is an extra issue to do it afterwards (which should be also be ready for next ff) https://github.com/elastic/kibana/issues/195182. Are we talking about the same?

Reply:
ah, I didn't realize there was a separate issue. Nevermind then!